frontend/src/App.js

```diff
@@ -14,84 +14,96 @@ import UCSBDatesEditPage from "main/pages/UCSBDates/UCSBDatesEditPage";
-        {
-          hasRole(currentUser, "ROLE_ADMIN") && (
-            <>
-              <Route exact path="/ucsbdates/edit/:id" element={<UCSBDatesEditPage />} />
-              <Route exact path="/ucsbdates/create" element={<UCSBDatesCreatePage />} />
-            </>
-          )
-        }
-
-        {
-          hasRole(currentUser, "ROLE_USER") && (
-            <>
-              <Route exact path="/ucsbsubjects/list" element={<UCSBSubjectsIndexPage />} />
-            </>
-          )
-        }
-        {
-          hasRole(currentUser, "ROLE_ADMIN") && (
-            <>
-              <Route exact path="/ucsbsubjects/edit/:id" element={<UCSBSubjectsCreatePage />} />
-              <Route exact path="/ucsbsubjects/create" element={<UCSBSubjectsCreatePage />} />
-            </>
-          )
-        }
-
+        {hasRole(currentUser, "ROLE_ADMIN") && (
+          <Route exact path="/admin/users" element={<AdminUsersPage />} />
+        )}
+        {hasRole(currentUser, "ROLE_USER") && (
+          <>
+            <Route exact path="/todos/list" element={<TodosIndexPage />} />
+            <Route exact path="/todos/create" element={<TodosCreatePage />} />
+            <Route
+              exact
+              path="/todos/edit/:todoId"
+              element={<TodosEditPage />}
+            />
+          </>
+        )}
+
+        {hasRole(currentUser, "ROLE_USER") && (
+          <>
+            <Route
+              exact
+              path="/students/list"
+              element={<StudentsIndexPage />}
+            />
+          </>
+        )}
+        {hasRole(currentUser, "ROLE_ADMIN") && (
+          <>
+            <Route
+              exact
+              path="/students/create"
+              element={<StudentsCreatePage />}
+            />
+          </>
+        )}
+
+        {hasRole(currentUser, "ROLE_USER") && (
+          <>
+            <Route
+              exact
+              path="/ucsbdates/list"
+              element={<UCSBDatesIndexPage />}
+            />
+          </>
+        )}
+        {hasRole(currentUser, "ROLE_ADMIN") && (
+          <>
+            <Route
+              exact
+              path="/ucsbdates/edit/:id"
+              element={<UCSBDatesEditPage />}
+            />
+            <Route
+              exact
+              path="/ucsbdates/create"
+              element={<UCSBDatesCreatePage />}
+            />
+          </>
+        )}
+        {hasRole(currentUser, "ROLE_USER") && (
+          <>
+            <Route
+              exact
+              path="/collegiateSubreddits/list"
+              element={<CollegiateSubredditsIndexPage />}
+            />
+          </>
+        )}
+        {hasRole(currentUser, "ROLE_ADMIN") && (
+          <>
+            <Route
+              exact
+              path="/collegiateSubreddits/create"
+              element={<CollegiateSubredditsCreatePage />}
+            />
+          </>
+        )}
       </Routes>
     </BrowserRouter>
   );
```

Comment:
revert `App.js` commits and then add your changes back in (ie don't delete anything)

Reply:
Done! Good catch